File: gradio_api/app_api.py
from gradio_client import Client

#client = Client("http://127.0.0.1:7860/")
#result = client.predict(
##		"Jim!!",	# str  in 'name' Textbox component
#		api_name="/predict"
#)

from gradio_client import Client

client = Client("http://127.0.0.1:7860/")
result = client.predict(
		"camerio!!",	# str  in 'Name' Textbox component
		api_name="/greet"
)
#print(result)
print(result)

File: gradio_api/app_api.js
import { createClient } from "@gradio/client";

async function main() {
    const app = await createClient("http://127.0.0.1:7860/");
    const result = await app.predict("/greet", [		
        "jim ready!!", // string  in 'Name' Textbox component
    ]);

    console.log(result.data);
}

main();
